src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.IncrementalAnalyzerProcessor.cs

```diff
@@ -200,6 +206,24 @@ static async Task AnalyzeSyntaxAsync(IIncrementalAnalyzer analyzer, TextDocument
                             await analyzer2.AnalyzeNonSourceDocumentAsync(textDocument, reasons, cancellationToken).ConfigureAwait(false);
                         }
                     }
+
+                    async Task<bool> ProcessActiveDocumentSwitchedAsync(ImmutableArray<IIncrementalAnalyzer> analyzers, WorkItem workItem, TextDocument document, CancellationToken cancellationToken)
+                    {
+                        try
+                        {
+                            if (!workItem.InvocationReasons.Contains(PredefinedInvocationReasons.ActiveDocumentSwitched))
+                            {
+                                return false;
+                            }
+
+                            await RunAnalyzersAsync(analyzers, document, workItem, (a, d, c) => a.ActiveDocumentSwitchedAsync(d, c), cancellationToken).ConfigureAwait(false);
```

Comment:
💡 Better names for a, d, c would help

Reply:
I am following the existing naming used throughout this document :-)

Reply:
Let me clean up the names in this whole document.